src/main/scala/com/typesafe/sbt/packager/jdkpackager/JDKPackagerHelper.scala

```diff
@@ -62,40 +42,79 @@ object JDKPackagerHelper {
     basename: String,
     iconPath: Option[File],
     outputDir: File,
-    sourceDir: File) = {
+    sourceDir: File): Map[String, String] = {
 
-    def iconArg = iconPath
+    val iconArg = iconPath.toSeq
       .map(_.getAbsolutePath)
-      .map(p ⇒ Map(s"-Bicon=$p" -> ""))
-      .getOrElse(Map.empty)
+      .map(p ⇒ s"-Bicon=$p")
 
-    def mainClassArg = mainClass
+    val mainClassArg = mainClass
       .map(c ⇒ Map("-appclass" -> c))
       .getOrElse(Map.empty)
 
-    //    val cpSep = sys.props("path.separator")
-    //    val cp = classpath.map(p ⇒ "lib/" + p)
-    //    val cpStr = cp.mkString(cpSep)
+    // Make a setting?
+    val jvmOptsFile = (sourceDir ** "jvmopts").getPaths.headOption.map(file)
```

Comment:
>  but I worked hard to avoid introducing any new constructs here
And you have done a great job here :) 

The application configuration has a real big scope. Hopefully there is some time
to finish the discussion in #82 and find a proper solution. I like your idea of generating
the `jvmopts` or `etc-default` files from build settings.